### PR TITLE
Auto-add gamm LP to the assets

### PR DESCRIPTION
**Pull request by Buckram123**
Not sure if that should have been done through typescript, but that worked :)

### CODE REVIEW

File: cron/src/pools.rs [hunk 1]
```diff
@@ -22,20 +24,39 @@ pub fn get_scraped_entries(
...
+                    .1
+                    .assets
+                    .iter()
+                    .map(AssetEntry::as_str)
+                    .collect::<Vec<_>>()
+                    .join(",");
+                let asset_name = format!("{dex}/{assets}",);
+                lp_assets.insert(
+                    asset_name,
+                    AssetInfoBase::Native(format!("gamm/pool/{pool_id}")),
```
Comment (cyberhoward) [inline cron/src/pools.rs:47]: this will do this for every chain, we only want this for osmosis right?

Comment (Buckram123) [reply cron/src/pools.rs:47]: Pool ids is only osmosis right now

Comment (Buckram123) [reply cron/src/pools.rs:47]: we can do it only for osmosis, if we think other chains that add that type of module will use not gamm

Comment (cyberhoward) [reply cron/src/pools.rs:47]: I don't see where you only filter for Osmosis to register this data

Comment (Buckram123) [reply cron/src/pools.rs:47]: > I don't see where you only filter for Osmosis to register this data

Do we need to? Pool ids are currently available only on osmosis(gamm module). I will add filter if we expect other chain to introduce their own "pool module"

Comment (cyberhoward) [reply cron/src/pools.rs:47]: I don't see why we would need to register the LP tokens of astroport with a key of osmosis's gamm module.

**Pull request merged**